src/reps/load-reps.js

```diff
@@ -13,27 +13,15 @@ define(function (require, exports, module) {
   let parseURLEncodedText;
   let parseURLParams;
 
-  // useRepsBundle hardcoded to false while we experiment with the reps bundle in
-  // bugs 1325401 and 1330779. Set it to true to test devtools using the reps bundle.
-  let useRepsBundle = false;
```

Comment:
hmm, I'd probably just flip this to `true` as when we update, we might break things and it's nice to be able to flip the flag to `false` again